scripts/yaml_check.py

```diff
@@ -0,0 +1,41 @@
+import yaml
+import sys
+
+def tiered_extract(db):
+  out = []
+  for _, entry in db.items():
+    for _, values in entry.items():
+      for name, decl in values.items():
+        if decl is not None and decl != '' and '/' not in decl:
+          out.append((name, decl))
+  return out
+
+def print_list(fn, pairs):
+  with open(fn, 'w') as out:
+    for (id, val) in pairs:
+      out.write(f'{id}\n{val.strip()}\n\n')
+
+hundred_yaml = sys.argv[1]
+overview_yaml = sys.argv[2]
+undergrad_yaml = sys.argv[3]
```

Comment:
Why not hardcoding those names? Changing the command line call in the CI script won't be easier than changing this script, right?

Reply:
Just to avoid remembering how to use pathlib or hard-coding a particular file path format. The script can be tested locally on Windows like this. I think it doesn't matter much.